charts/premium/authelia/templates/_configmap.tpl

```diff
@@ -245,9 +245,11 @@ data:
     {{- range $client := .Values.identity_providers.oidc.clients }}
         - id: {{ $client.id }}
           description: {{ $client.description | default $client.id }}
-          secret: {{ $client.secret | default (randAlphaNum 128) }}
           {{- if $client.public }}
           public: {{ $client.public }}
+          secret: ""
+          {{- else }}
+          secret: {{ $client.secret  }}
```

Comment:
still needs a failcase

Reply:
dunno how to replicate `authelia crypto hash generate pbkdf2 --variant sha512 --random --random.length 72 --random.charset rfc3986` to be set as failcase in helmchart though :sweat_smile: